include/RAJA/Timer.hxx

```diff
@@ -91,7 +91,7 @@ public:
                  + std::chrono::microseconds(tstart.tv_usec);
     auto stop = std::chrono::seconds(tstop.tv_sec)
                 + std::chrono::microseconds(tstop.tv_usec);
-    telapsed += std::chrono::duration<double>(stop - start).count();
+    telapsed += std::chrono::duration<long double>(stop - start).count();
```

Comment:
The Caliper garbage is mine, that should definitely either not exist or be in a different class, a lot of this seems iffy, I'd just write it how you think it should look.